pulp_rpm/app/rpm_version.py

```diff
@@ -0,0 +1,243 @@
+# Sourced from https://github.com/nexB/univers
```

Comment:
Vendor, or pull the whole huge package?  It supports version comparisons for a dozen different package types (arch, debian, python, ruby, go, etc.)

Reply:
My take would be to just take the code in-line, rather than dragging a huge surface into pulp_rpm most of which we do not need.